offset_manager_test.go

```diff
@@ -228,6 +228,33 @@ func TestPartitionOffsetManagerMarkOffset(t *testing.T) {
 	coordinator.Close()
 }
 
+func TestPartitionOffsetManagerMarkOffsetWithRetention(t *testing.T) {
```

Comment:
This test doesn't actually test that the value in the request is set correctly. It should do that, or it should be removed.